internal/graph/graph.go

```diff
@@ -196,6 +196,9 @@ func (g *ConnectedObjectGraph) findIngressesWithRewrite(
 		for _, typeRestriction := range tuplesetTypeRestrictions {
 
 			r, err := g.typesystem.GetRelation(typeRestriction.GetType(), computedUserset)
+			if err != nil {
+				continue // skip computed relations on tupleset relationships if they are undefined
```

Comment:
We should probably be more specific and check specifically on `typesystem.ErrRelationUndefined`

```
if errors.Is(err, typesystem.ErrRelationUndefined) {
    continue
}

return nil, err
```

Reply:
Otherwise we might blindly skip something that should have led to an error otherwise.

Reply:
Good point! Updated. Thanks!